src/models/Review.py

```diff
@@ -0,0 +1,31 @@
+#temp
+from bson import ObjectId
+import datetime
+
+class Review:
+  _id: ObjectId
+  user_id: str
+  drink_id: str
+  comment: set[str] = set() #comments of drinks from this review
+  date: datetime.datetime.now()
```

Comment:
I'm not quite sure if this code does what you were planning for it to do. This will set the type of `date` to what's returned from `datetime.datetime.now()`. This is fine, but `self.date` would be undefined. I think what you might be looking for is replacing the colon with `=`.